xrpl4j-model/src/test/java/org/xrpl/xrpl4j/model/transactions/TransactionTypeTests.java

```diff
@@ -0,0 +1,60 @@
+package org.xrpl.xrpl4j.model.transactions;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.ArgumentsProvider;
+import org.junit.jupiter.params.provider.ArgumentsSource;
+import org.junit.jupiter.params.provider.EmptySource;
+import org.junit.jupiter.params.provider.NullSource;
+
+public class TransactionTypeTests {
+
+	public static class TransactionTypeValidArgumentProvider implements ArgumentsProvider {
+
+		@Override
+		public Stream<? extends Arguments> provideArguments(ExtensionContext context) throws Exception {
+			return Stream.of(TransactionType.values()).map(TransactionType::value).map(Arguments::of);
+		}
+
+	}
+
+	public static class TransactionTypeInvalidArgumentProvider implements ArgumentsProvider {
+
+		@Override
+		public Stream<? extends Arguments> provideArguments(ExtensionContext context) throws Exception {
+			return Stream.of("bla", "blaaa","123").map(Arguments::of);
+		}
+
+	}
+
+	@ParameterizedTest
+	@ArgumentsSource(value = TransactionTypeValidArgumentProvider.class)
+	public void shouldReturnTransactionTypeForValidValues(String value) {
+		System.out.println("running test shouldReturnTransactionTypeForValidValues with argument " + value);
+
+		TransactionType transactionType = TransactionType.forValue(value);
+		assertNotNull(transactionType);
+		assertTrue(transactionType instanceof TransactionType);
+		System.out.println("=========================================OK==============================================");
+	}
+
+	@EmptySource
+	@NullSource
+	@ParameterizedTest
+	@ArgumentsSource(value = TransactionTypeInvalidArgumentProvider.class)
+	public void shouldThrowIllegalArgumentExceptionForInvalidValues(String value) {
+		System.out.println("running test shouldThrowIllegalArgumentExceptionForInvalidValues with argument " + value);
```

Comment:
Let's either use a `Logger` here or instead just consider removing the emissions. For example, are these emissions adding any value to the actual build? Seems like not because if a test fails, the assertion will indicate what went wrong down on line 55.

Reply:
You are absolutely right.I will remove it.